### PR TITLE
[v6r13] Fix to use cfgPath

### PR DESCRIPTION
**Pull request by rajanandakumar**
I hope this is correct usage. I have not been able to test this minor patch as the test system is under flux right now.

### CODE REVIEW

File: WorkloadManagementSystem/Service/WMSUtilities.py [hunk 1]
```diff
@@ -78,8 +79,8 @@ def getARCPilotOutput( proxy, pilotRef ):
   myce = pilotRef.split(":")[1].strip("/")
   gridEnv = getGridEnv()
   mySite = getCESiteMapping()['Value'][myce]
-  WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))
-  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL2", WorkDB)
+  WorkDB = gConfig.getValue(cfgPath['Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'])
```
Comment (sposs) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: cfgPAth is a function, you are missing ( and ) around the list.

File: WorkloadManagementSystem/Service/WMSUtilities.py [hunk 2]
```diff
@@ -78,8 +79,8 @@ def getARCPilotOutput( proxy, pilotRef ):
...
+  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL", WorkDB)
```
Comment (fstagni) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: It was a sort of hack before, and this is not better. Can you make something generic? Do you need this specific directory?

Also, minor comments: 
- variables should be in camelCase with first letter lowercase, so use "workDB" not "WorkDB".
- there's a "print" few lines below, please remove it
- are lines 103-107 needed?

File: WorkloadManagementSystem/Service/WMSUtilities.py (hunk 1 above)
Comment (rajanandakumar) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: Thanks - will fix that.

File: WorkloadManagementSystem/Service/WMSUtilities.py (hunk 2 above)
Comment (rajanandakumar) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: Indeed this is a temporary "fix", and something more "generic" is pretty much impossible using the CLI. I only hope that this code will not break anything. If pilots work from one site, even occasionally, it is a success.

Happy to explain this next week in person.

Comment (rajanandakumar) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: And yes - lines 103 - 107 are needed. They are the ones which actually return the pilot log to the screen.

File: WorkloadManagementSystem/Service/WMSUtilities.py [hunk 3]
```diff
@@ -78,12 +79,11 @@ def getARCPilotOutput( proxy, pilotRef ):
   myce = pilotRef.split(":")[1].strip("/")
   gridEnv = getGridEnv()
   mySite = getCESiteMapping()['Value'][myce]
-  WorkDB = gConfig.getValue(os.path.join('Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile'))
-  myWorkDB = os.path.join("/opt/dirac/runit/WorkloadManagement/SiteDirector-RAL2", WorkDB)
+  workDB = gConfig.getValue(cfgPath(['Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile']))
```
Comment (atsareg) [inline WorkloadManagementSystem/Service/WMSUtilities.py]: This line must be just:
workDB = gConfig.getValue( cfgPath( 'Resources/Sites/LCG', mySite, 'CEs', myce, 'JobListFile' ) )

**Comment by rajanandakumar** [comment]
Thanks! Updated the code ...

**Pull request merged**